docker/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM debian:8.2
```

Comment:
We'd need #365 implemented first - the current binary has too many dynamic links to be easily usable in a minimal image. Ideally the docker image would be built and pushed as part of CI, I'd like to get that working next. With those both done it'd be very easy to make a minimal image: write a dockerfile that just adds the built binary into it during CI, no download necessary.